src/displayBackend/drm/Display.hpp

```diff
@@ -51,7 +51,7 @@ class Display : public DisplayItf::Display
 	/**
 	 * @param name device name
 	 */
-	explicit Display(const std::string& name);
+	explicit Display(const std::string& name, bool disable_zcopy = false);
```

Comment:
explicit is not required

Reply:
task for refactoring, won't fix now, see the same above

Reply:
Filed a bug #93

Reply:
https://www.modernescpp.com/index.php/c-core-guidelines-constructors
Rule C.46